_includes/pbc/all/install-features/202304.0/install-the-picking-app-feature.md

```diff
@@ -0,0 +1,173 @@
+This document describes how to integrate the Picking App feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Picking App feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features and Glue APIs:
+
+| NAME                     | VERSION          |
+|--------------------------|------------------|
+| Spryker Core             | {{site.version}} |
+| Spryker Core Back Office | {{site.version}} |
+| Inventory Management     | {{site.version}} |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker-feature/picking-app: "{{site.version}}" --update-with-dependencies
```

Comment:
Lets clarify mapping with PO and AA again, Warehouse User modules belongs to Inventory Management feature or sub-feature of Inventory Management.
Imo name could be - `spryker-feature/warehouse-user-management`